### ISSUE TITLE
Arrow icon on variant selector drop-down is too large

### ISSUE THREAD

**Issue by Joe-Brady** [AdoptOpenJDK/openjdk-website#189]
* **Browser**: all

The arrow symbol on the drop-down list of variants is too big.

To change it, the below CSS selector needs to be changed to have a 90%/1rem value, as shown (the 1rem part controls the size of the arrow - it was previously set to a percentage, which is not suitable - my bad!):

```
select#platform-selector, select#variant-selector {
    background: url(../assets/arrow-down.png) 90%/1rem no-repeat #fff;
}
```

**Comment by EnriqueL8** [comment]
okay , I'll change it now

**Comment by gdams** [comment]
@EnriqueL8 any idea if this change was made?

**Comment by karianna** [comment]
I've fixed this in #321
